modules/lightning_features/lightning_core/modules/lightning_search/css/entity-search.css

```diff
@@ -0,0 +1,3 @@
+.ui-autocomplete-loading[data-entity-search] {
```

Comment:
The core autocomplete library brings in that basic styling, plus a lot of dross that we can't use (I know because I tried). This was the path of least resistance. I don't think we can import assets libraries we don't "own"...